wazuh/wazuh-dashboard-plugins
# Fix vulnerabilities dashboard custom KPIs labels

### Description
This pull request fixes the vulnerabilities KPI's custom labels so they properly add the filter label when the user clicks them
 
### Issues Resolved
Parent issue:
- https://github.com/wazuh/wazuh-dashboard-plugins/issues/6533

### Evidence
![Peek 2024-03-27 14-36](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/09cb52c5-8702-4cdf-bb4f-6027b2ce5371)


### Test
- Go to Vulnerabilities dashboard
- Check the KPI's label are `Critical - Severity` `High - Severity` `Medium - Severity` `Low - Severity`
- Click on the KPI's and check they add the custom label filter with the severity value

### Check List
